quick-start/src/main/resources/application.properties

```diff
@@ -1,10 +1,11 @@
 #Spring embedded tomcat config. Does not apply when war file is deployed on external Tomcat.
-server.port=8080
+server.port=8081
 server.contextPath=/
 spring.thymeleaf.cache=true
 logging.level.com.marklogic.hub: DEBUG
 mlHost.default=localhost
-mlRestPort.default=8010
+mlStagingRestPort.default=8010
+mlFinalRestPort.default=8011
 mlUsername.default=admin
```

Comment:
Looks like this username is used to configure the hub. If it's also going to be used for application access, are there situations in which staging and final (different app servers) should use different credentials?